src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/service/PrisonerSearchApiService.kt

```diff
@@ -10,11 +10,12 @@ private val log = KotlinLogging.logger {}
 @Service
 class PrisonerSearchApiService(
   private val prisonerSearchApiClient: PrisonerSearchApiClient,
+// prisoner-search-api appears to be buggy in it's paged API implementation, so safest to request a large page size to get all records in one hit (arguably not the most efficient though)
```

Comment:
This is the core fix - set a default initial value of 9999 to get us out of bother here and now, but its now a constructor arg so we can set different values as and when the time comes